app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -551,4 +600,105 @@ protected void onDestroy() {
         super.onDestroy();
         mapView.onDestroy();
     }
+
+    /**
+     * Zooms in or out of a card depending on the argument
+     * @param zoomId "zoom_in" to zoom in, "zoom_out" otherwise
+     */
+    private void zoom(String zoomId){
```

Comment:
For the codeClimate warning. It is true that GuessLocationActivity is starting to be huge so if it is possible to put the zoom functions into another helper file without too much work it would be great.

Reply:
I put the zoom in another class, even though I'm not sure my zoom animation was the main problem regarding this activity's code clarity.